src/display/editor/toolbar.js

```diff
@@ -36,6 +36,21 @@ class EditorToolbar {
     buttons.className = "buttons";
     editToolbar.append(buttons);
 
+    const position = this.#editor.toolbarPosition;
+    if (position) {
+      const { style } = editToolbar;
+      if (this.#editor._uiManager.direction === "ltr") {
+        style.left = "unset";
+        style.right = `${100 * (1 - position[0])}%`;
+      } else {
+        style.left = `${100 * position[0]}%`;
+        style.right = "unset";
+      }
```

Comment:
I'm assuming that you've already tried, but could we possibly use `inset-inline-start` or `inset-inline-end` rather than explicitly setting `left/right` here?